lib/presentation/widget/shared_space/file_surfing/workgroup_nodes_surfing_widget.dart

```diff
@@ -489,16 +490,28 @@ class _WorkGroupNodesSurfingWidgetState extends State<WorkGroupNodesSurfingWidge
         .build();
   }
 
-  Widget _copyToMySpaceAction(BuildContext context, List<WorkGroupNode> workGroupNodes) {
-    return workGroupNodes.any((element) => element is WorkGroupFolder) ?
-      SizedBox.shrink() :
-      WorkGroupDocumentContextMenuTileBuilder(
-        Key('copy_in_my_space_context_menu_action'),
-        SvgPicture.asset(imagePath.icCopy,
+  Widget _copyToAction(
+    BuildContext context,
+    List<WorkGroupNode> nodes,
+    {ItemSelectionType itemSelectionType = ItemSelectionType.single}
+  ) {
+    return nodes.any((element) => element is WorkGroupFolder)
+      ? SizedBox.shrink()
+      : WorkGroupDocumentContextMenuTileBuilder(
+          Key('copy_to_context_menu_action'),
+          SvgPicture.asset(
+            imagePath.icSharedSpace,
             width: 24, height: 24, fit: BoxFit.fill),
-        AppLocalizations.of(context).copy_to_my_space,
-        workGroupNodes.first)
-        .onActionClick((data) => _model.copyToMySpace(workGroupNodes))
+          AppLocalizations.of(context).copy_to, nodes[0])
+        .trailing(SvgPicture.asset(
+          imagePath.icArrowRight,
+          width: 24, height: 24,
+          fit: BoxFit.fill))
+        .onActionClick((data) => _model.copyTo(
+          context,
+          nodes,
+          [DestinationType.mySpace, DestinationType.workGroup],
```

Comment:
`destinationTypeList` can be initialized with shorter way like this: `DestinationType.values`

Reply:
but what if in the future we have more types in this enum but we only want to init this list with these 2 types, then using `DestinationType.values` will be hard to control.

Reply:
> but what if in the future we have more types in this enum but we only want to init this list with these 2 types, then using `DestinationType.values` will be hard to control I think

yes, in that case it's correct.